ncm-grub/src/main/perl/grub.pm

```diff
@@ -200,16 +200,17 @@ sub password
             owner => "root",
             group => "root",
             mode  => oct(400),
-            log   => $self );
+            log   => $self,
+            sensitive => 1 );
```

Comment:
remove the whitespce.
even better, add a `,` and move the `);` to the next line (so next option will have clean diff)